packages/flutter/lib/src/material/app.dart

```diff
@@ -163,9 +166,45 @@ class MaterialApp extends StatefulWidget {
   /// This value is passed unmodified to [WidgetsApp.onGenerateTitle].
   final GenerateAppTitle onGenerateTitle;
 
-  /// The colors to use for the application's widgets.
+  /// Default visual properties, like colors fonts and shapes, for this app's
+  /// material widgets.
+  ///
+  /// A second [darkTheme] [ThemeData] value, which is used when the underlying
+  /// platform requests a "dark mode" UI, can also be specified.
+  ///
+  /// The default value of this property is a light [ThemeData.fallback].
```

Comment:
This doesn't read very well. I think you mean: a light theme, the value of [ThemeData.fallback]. However [TheemData.fallback] doesn't seem like a useful value to mention, since it's hardwired to ThemeData.light().

So: The default value of this property is the value of [ThemeData.light()]. That's equally true and easier to understand.

Reply:
Done